tests/CMakeLists.txt

```diff
@@ -22,10 +22,73 @@ function(copy_test_data name)
     FILE_PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
 endfunction()
 
+
```

Comment:
Whenever you make signficant changes or additions, add your name to the list of authors at the top of the file.

Reply:
Just your name, not a new date. Just list your name after my name...